tests/compas/datastructures/test_mesh.py

```diff
@@ -49,12 +49,12 @@ def test_from_ply():
 
 
 def test_from_stl():
-    mesh = Mesh.from_stl(compas.get('cube_ascii.stl'))
+    mesh = Mesh.from_stl(compas.get('cube_ascii.stl'), '6f')
```

Comment:
still a question for me how do reliably deal with precision. this is btw, why i started adding header info to the files that we write... i think precision should be part of that...

Reply:
Headers sounds like a sound idea ;)  Additionally, the serialization formats used by compas should have a schema somewhere, and support versioning of themselves; but all that is a discussion for another PR.